algobot/helpers.py

```diff
@@ -90,65 +137,62 @@ def open_folder(folder: str):
     """
     This will open a folder even if it doesn't exist. It'll create one if it doesn't exist.
     """
-    targetPath = create_folder(folder)
-    open_file_or_folder(targetPath)
+    target_path = create_folder(folder)
+    open_file_or_folder(target_path)
 
 
-def create_folder(folder: str):
+def create_folder(folder: str) -> str:
     """
     This will create a folder if needed in the root directory.
     """
-    targetPath = os.path.join(ROOT_DIR, folder)
-    create_folder_if_needed(targetPath)
+    create_folder_if_needed(folder)
 
-    return targetPath
+    return folder
 
 
-def create_folder_if_needed(targetPath: str, basePath: str = ROOT_DIR) -> bool:
+def create_folder_if_needed(target_path: str) -> bool:
     """
     This function will create the appropriate folders in the root folder if needed.
-    :param targetPath: Target path to have exist.
-    :param basePath: Base path to start from. By default, it'll be the root directory.
+    :param target_path: Target path to have exist.
     :return: Boolean whether folder was created or not.
     """
-    if not os.path.exists(targetPath):
-        folder = os.path.basename(targetPath)
-        os.mkdir(os.path.join(basePath, folder))
+    if not os.path.exists(target_path):
+        os.makedirs(target_path)
```

Comment:
We can also use the exist_ok kwarg and set that to True so we don't need to check the conditional

Reply:
Won't that change the semantics of the function? 

As in we return true/false if the directory exists?